pocs/__init__.py

```diff
@@ -6,127 +6,11 @@
-    to be set in order for PANOPTES to work correctly. This method just
-    sanity checks our environment and shuts down otherwise.
-
-        PANDIR    Base directory for PANOPTES
-        POCS      Base directory for POCS
-    """
-    if sys.version_info[:2] < (3, 0):  # pragma: no cover
-        warn("POCS requires Python 3.x to run")
-
-    pandir = os.getenv('PANDIR')
-    if not os.path.exists(pandir):
-        sys.exit("$PANDIR dir does not exist or is empty: {}".format(pandir))
-
-    pocs = os.getenv('POCS')
-    if pocs is None:  # pragma: no cover
-        sys.exit('Please make sure $POCS environment variable is set')
-
-    if not os.path.exists(pocs):
-        sys.exit("$POCS directory does not exist or is empty: {}".format(pocs))
-
-    if not os.path.exists("{}/logs".format(pandir)):
-        print("Creating log dir at {}/logs".format(pandir))
-        os.makedirs("{}/logs".format(pandir))
-
-
-def _check_config(temp_config):
-    """ Checks the config file for mandatory items """
-
-    if 'directories' not in temp_config:
-        sys.exit('directories must be specified in config')
-
-    if 'mount' not in temp_config:
-        sys.exit('Mount must be specified in config')
-
-    if 'state_machine' not in temp_config:
-        sys.exit('State Table must be specified in config')
-
-
-_check_environment()
-
-# Global vars
-_config = None
-_logger = None
-
-
-class PanBase(object):
-
-    """ Base class for other classes within the Pan ecosystem
-
-    Defines common properties for each class (e.g. logger, config)self.
-    """
-
-    def __init__(self, *args, **kwargs):
-        # Load the default and local config files
-        global _config
-        if _config is None:
-            ignore_local_config = kwargs.get('ignore_local_config', False)
-            _config = config.load_config(ignore_local=ignore_local_config)
-
-        # Update with run-time config
-        if 'config' in kwargs:
-            _config.update(kwargs['config'])
-
-        _check_config(_config)
-        self.config = _config
-
-        global _logger
-        if _logger is None:
-            _logger = get_root_logger()
-            _logger.info('{:*^80}'.format(' Starting POCS '))
-
-        self.logger = kwargs.get('logger', _logger)
-
-        self.__version__ = __version__
-
-        if 'simulator' in kwargs:
-            if 'all' in kwargs['simulator']:
-                self.config['simulator'] = ['camera', 'mount', 'weather', 'night']
-            else:
-                self.config['simulator'] = kwargs['simulator']
-
-        # Set up connection to database
-        db = kwargs.get('db', self.config['db']['name'])
-        _db = PanMongo(db=db)
-
-        self.db = _db
-
-    def __getstate__(self):  # pragma: no cover
-        d = dict(self.__dict__)
-
-        if 'logger' in d:
-            del d['logger']
-
-        if 'db' in d:
-            del d['db']
-
-        return d
-
-
+from .version import __version__
```

Comment:
Please use root relative paths.

Reply:
This might be an odd situation where that doesn't work as it is needed for setup.py, which means it's needed before module is installed. I'll double check and change if not